core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorSpec.scala

```diff
@@ -2328,14 +2315,28 @@ class SubmissionMonitorSpec(_system: ActorSystem)
 
     withWorkspaceContext(manySubmissionsTestData.workspace) { ctx =>
       val indiv1 = runAndWait(
-        entityQuery.get(ctx, legacyTestData.indiv1.entityType, legacyTestData.indiv1.name)
-      ).get
+        compactEntityQuery.getEntity(ctx.workspaceIdAsUUID,
+                                     legacyTestData.indiv1.entityType,
+                                     legacyTestData.indiv1.name
```

Comment:
If you've switched to compactEntityQuery, this should just be testData, not legacyTestData

Reply:
should it be `testData` or `manySubmissionsTestData`?

Reply:
hm, well, `testData` is the only one that compiles

Reply:
Hm looks like [originally](https://github.com/broadinstitute/rawls/commit/6d908d783c40912524a98c12cb09a65757e35163) it was `testData`.  I guess `manySubmissionsTestData` was additional data on top of `testData`, so the specifically defined entities were just in `testData`.